packages/client/hooks/useMenu.ts

```diff
@@ -53,11 +63,30 @@ const useMenu = <T extends HTMLElement = HTMLButtonElement>(
     return Math.max(40, bbox ? bbox.width : 40)
   }, [originRef.current /* eslint-disable-line react-hooks/exhaustive-deps */])
   const {loadingDelay, loadingDelayRef} = useLoadingDelay()
+
+  const [topOffset, setTopOffset] = useState<number>(0)
+
+  const menuCoords = useMemo(() => {
+    if (
+      portalStatus === PortalStatus.Mounted ||
+      portalStatus === PortalStatus.Entering ||
+      portalStatus === PortalStatus.Entered
```

Comment:
+1 this could better simplify to `[PortalStatus.Mounted, PortalStatus.Entering, PortalStatus.Entered].includes(portalStatus)`